### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by brunoolivieri**

### CODE REVIEW

File: common/source/docs/common-electro-permanent-magnet-V3.rst [hunk 1]
```diff
@@ -57,7 +57,7 @@ Firmware setup
```
Comment (Hwurzburg) [inline common/source/docs/common-electro-permanent-magnet-V3.rst]: I suggest removing this entire section....UAVCAN is supported now....3.4 is old and doesnt need mentioning....

**Comment by rmackay9** [comment]
As discussed, let's remove the firmware section of this page.

**Comment by brunoolivieri** [comment]
Updated and rebased.

**Comment by Hwurzburg** [comment]
LGTM...merging

**Pull request merged**